features/docs/writing_support_code/after_hooks.feature

```diff
@@ -11,7 +11,7 @@ Feature: After Hooks
   Background:
     Given the standard step definitions
 
-  Scenario Outline: Retreive the status of a scenario as a symbol
+  Scenario Outline: Retrieve the status of a scenario as a symbol
```

Comment:
Thanks. I know how tempting it is to fix little things like this "while you're there" (and I very much appreciate you doing it!) but it makes it much easier for us to read / review a PR if it's focussed on just one change.